src/main/java/teammates/ui/controller/Action.java

```diff
@@ -672,4 +676,22 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
     protected void excludeStudentDetailsFromResponseParams() {
         regkey = null;
     }
+
+    protected void verifyAccessibleForInstructorToFeedbackResponseComment(
```

Comment:
This should not be in the `Action` class, since this method only applies to two classes, as opposed to the majority of classes that inherit from `Action`. If inheritance is necessary, create a new abstract class extending `Action` with the shared method, and have `InstructorFeedbackResponseCommentDeleteAction` and `InstructorFeedbackResponseCommentEditAction` extend that abstract class (refer to `InstructorCourseInstructorAbstractAction` for an example).